### ISSUE TITLE
Safari extension

### ISSUE THREAD

**Issue by Raph33** [padloc/padloc#293]
Is there a plan to release an extension for the web browser Safari?

**Comment by MaKleSoft** [comment]
Absolutely! Since starting with Big Sur Safari will support the Web Extensions standard, distributing the Extension for Safari 14 and up should be fairly easy. So once Big Sur is out of beta (and I've had a chance to install it) this will be one of the first things on the list.
